CHANGELOG.md

```diff
@@ -1,21 +1,35 @@
 # Changelog
 
+## v1.5.0
+
+This release updates the Linux kernel from 4.4 to 4.14.
+
+* Enhancements
+  * Added Alsa utils `aplay` and `amixer` for audio support.
+  * Added `espeak` a speech synthesizer for text to audio.
+  * Reboot if Linux kernel panics after 10 second delay.
```

Comment:
Automatically reboot if the Linux kernel panics (defaults to a 10 second delay before the reboot)